lib/DonorFilter/PluggableDonorFilter.js

```diff
@@ -0,0 +1,121 @@
+import { map, sortBy } from 'lodash';
+import React, {
```

Comment:
Starting from version 17, it is unnecessary to import `React` globally in each file to support `JSX`. React and Babel libs do it automatically.